pkg/controller/kogitoapp/resource/requested_resources_test.go

```diff
@@ -154,7 +160,34 @@ func Test_buildConfigS2IBuilder(t *testing.T) {
 
 	chain := buildConfigS2IBuilder(blankBuildSpec)
 	assert.NotNil(t, chain)
-	assert.Nil(t, chain.Resources.BuildConfigS2I)
+	assert.NotNil(t, chain.Resources.BuildConfigS2I)
+
+	chain = buildConfigS2IBuilder(gitURLGiven)
+	assert.NotNil(t, chain)
+	assert.NotNil(t, chain.Resources.BuildConfigS2I)
+}
+
+func Test_buildConfigS2IBuilderWithoutGitUrl(t *testing.T) {
```

Comment:
It seems that this test is same as Test_buildConfigS2IBuilderWithGitUrl

Reply:
basically yes, but one has no giturl defined. so, both scenarios, must have the same buildConfigs getting created.